.gitlab-ci.yml

```diff
@@ -12,10 +21,18 @@ variables:
 
 # Test Rust template only.
 test-rust:
+    variables:
+      ZOOKEEPER_ENDPOINTS: "127.0.0.1:2181"
     script:
+    - apt-get update
+    - apt-get install --assume-yes zookeeper
```

Comment:
I just pushed a new docker image with zookeeper pre-installed, so it should be ok to remove these commands now

Reply:
Awesome! Thank you!